HELP.md

```
@@ -0,0 +1,29 @@
+# Read Me First
+The following was discovered as part of building this project:
+
+* The original package name 'com.miche-let.api-gateway' is invalid and this project uses 'com.miche_let.api_gateway' instead.
+
+# Getting Started
+
+### Reference Documentation
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)